sensor/Dockerfile.template

```diff
@@ -19,5 +19,6 @@ RUN pip install smbus
 WORKDIR /usr/src/app
 COPY ./scripts ./scripts
 COPY ./*.sh /usr/src/app/
+RUN chmod +x /usr/src/app/entry.sh
```

Comment:
Can work, but you can omit this line, do `chmod +x ...` on your computer and commit new file attributes only?

Reply:
@zrzka the file already has +x attributes, but in my experience this doesn't cover all cases. Not sure if it's users pushing from Windows or something but without this we always get issues.

Reply:
@chrisys ah, interesting, never had this issue (okay, not using Windows :), thanks for the explanation

Reply:
I personally have never been able to reproduce it (macOS user) but it's come up across all of the projects we promote heavily, might be an edge case.